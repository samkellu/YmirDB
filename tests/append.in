append a 1 2
set a 123 33 554
set b 11 a 46 22
get a
push a b
set c 123 334 b
append a 44 22 c
get a
bye
