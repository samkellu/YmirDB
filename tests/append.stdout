==2781== Memcheck, a memory error detector
==2781== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2781== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==2781== Command: ./ymirdb
==2781== 
==2781== error calling PR_SET_PTRACER, vgdb might block
> ok

> ok

> [123 33 554]

> ok

> ok

> ok

> [123 33 554 44 22 c]

> bye
==2781== 
==2781== HEAP SUMMARY:
==2781==     in use at exit: 0 bytes in 0 blocks
==2781==   total heap usage: 53 allocs, 53 frees, 2,904 bytes allocated
==2781== 
==2781== All heap blocks were freed -- no leaks are possible
==2781== 
==2781== For lists of detected and suppressed errors, rerun with: -s
==2781== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
